lib/erlef/blog/post.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Erlef.Blog.Post do
+  @moduledoc """
+  Schema for blog posts.
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "blog_posts" do
+    field(:title, :string)
+    field(:slug, :string)
+    field(:body, :string)
+    field(:status, Ecto.Enum, values: [:draft, :published, :archived])
+
+    field(:member_id, :id)
```

Comment:
I think both of the `:member_id` field and `:working_group_id` should be `belongs_to`. Maybe there was a reason not to go this route though? This if of course devoid of what we require.